File: CS1B/character_count.py
def main():
	print ("--------------------");
	print ("Counting characters");
	print("----------------------");
	print;
	string = raw_input("Enter anything(press Enter to finish typing): ");
	print string;
	count = 0;
	
	for ch in string:
		if ch == " ":
			continue;
		count += 1;
		
	print "Number of characters in \"%s\" is: %d character(s) without spaces" % (string, count);
	close = raw_input();
	
	
if __name__ == "__main__":
	main()

File: CS1B/character_count.c
#include<stdio.h>
#include<conio.h>
void main(void)
{
    char alph;
	int num;
	printf("Enter anything: ");
	for(num=0;;num++)
	{
		alph = getche();
		if(alph == 13)
		{break;}
	}
	if(num > 1)
	{printf("\nThere are %d characters.",num);}
	else if(num == 1)
	{printf("\nThere is only %d character.",num);}
	getch();
}


